contracts/src/v0.8/vrf/dev/VRFV2PlusWrapper.sol

```diff
@@ -420,11 +470,16 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
     // (wei/gas) * gas
     uint256 wrapperCostWei = _requestGasPrice * s_wrapperGasOverhead;
 
+    // calculate L1 gas fee cost for the calldata (via upper bound calculation or in the accurate way)
+    uint256 l1CalldataCost = (s_upperBoundLimitL1FeeCalculationEnabled)
```

Comment:
We might need to figure out a way to guard this storage access in a is l2 check as it is penalizing every other chain with 2100 gas cost.

Reply:
This part is changed now and only L2s should be affected